src/main/java/creativei/controller/ReminderController.java

```diff
@@ -41,4 +43,21 @@ ResponseObject createReminder(@RequestBody String reminderString, HttpServletReq
             return ResponseObject.getResponse(ExceptionType.GENERAL_ERROR.getMessage(), ExceptionType.GENERAL_ERROR.getCode());
         }
     }
```

Comment:
Use Get request

Reply:
Then how to use json for input param?

Reply:
Check spring docs

Reply:
changes to get request with input param as JSON in commit 3fe67e5